crates/core/src/vm.rs

```diff
@@ -518,15 +533,15 @@ impl<'db, 'tx> DbProgram<'db, 'tx> {
                         }
                     }
                 }
-                Ok(Code::Pass)
+                Ok(Code::Pass(None)) // TODO going to assume dropping a table doesn't need to broadcast its entire contents as a notify
```

Comment:
Assumption here is deleting a table a reducer/subscriber depends on is either undefined behavior or breaking things up.

Reply:
Correct; modifying the schema which clients depend on is unsupported.